components/nexus-core/src/main/java/org/sonatype/nexus/internal/SonarExample.java

```diff
@@ -0,0 +1,12 @@
+package org.sonatype.nexus.internal;
+
+/**
+ * Created by jason on 7/9/15.
+ */
+public class SonarExample
+{
+  public SonarExample() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Hide this public constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1118)